Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,25 @@
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script>
+    (function() {
+        var recommendationSupport = new eZ.RecommendationSupport();
+        recommendationSupport.setOptions({
+            limit: {{ limit }},
+            msgEmpty: '{{ 'No recommendations found'|trans }}',
```

Comment:
I would put `{% autoescape 'js' %}` just after the `<script>` tag and `{% endautoescape %}` just before `</script>`.